app/src/main/AndroidManifest.xml

```diff
@@ -30,6 +30,22 @@
 
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
+            <intent-filter>
+                <action android:name="android.intent.action.VIEW" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+
+                <!-- Accepts URIs that begin with "https://chat.susi.ai/app" -->
+                <data android:host="chat.susi.ai" android:pathPrefix="/app" android:scheme="https" />
+            </intent-filter>
+            <intent-filter>
+                <action android:name="android.intent.action.VIEW" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+
+                <!-- Accepts URIs that begin with "susi.ai://" -->
+                <data android:scheme="susi.ai" />
+            </intent-filter>
```

Comment:
This intent filter is too generic, we are not handling requests internally, so this will be useless.

Reply:
Any suggestion for URI? App should support deep linking, right?

Reply:
Yes, but until it does not, we can't add this. So, remove this intent-filter.

Reply:
2nd should be removed. The problem in registering it before the app has deep linking support is that users will be confused and file issues regarding it. Even now users file multiple issues on single bug

Reply:
Updated PR